### ISSUE TITLE
Project List window - Can't Create New Project

### ISSUE THREAD

**Issue by comtineo01** [xtuple/xtuple#2238]
New user - Manually entering Projects from 2014 (were jobs in QuickBooks) in order to GJ the costs into them. After entering a couple from the top level CRM>Projects>New, I realized it was much faster to leave the Projects List up and just click the 'New' button. However after 17 of them, xTuple will no longer allow this. When I click the 'New' button on the Projects List window, it comes up pre-filled out and even if I edit the info to the new project I want to enter - it will not save it. 

If I go back to the old method using the top level CRM>Projects>New, it seems to work fine but....
1) With many projects still to enter, it seems rather tedious.
2) I'm worried that any further work may be moot if the database is broken already after only a few new entries were made.

I received a helpful reply on the xTuple forum to try the System>Utilities > Access Control and Serial Columns, but that didn't seem to fix the issue.

**Comment by szuke** [comment]
I just added 20 projects without an issue. I assure you that the database isn't broken. I'm making an assumption that you're talking about the Qt client? I have been using Projects for years and have never encountered these type of issues. In addition, I've converted Job data from QuickBooks into Projects without an issue.

**Comment by andersdd** [comment]
Not a solution - but will solve the tedious issue - if you go to System Preferences you can define Hot keys where you can link something like the F1 key to the new Projects screen.

At least until we get clarification on what the issue might be.

**Comment by comtineo01** [comment]
Thank you for a productive reply andersdd. Noobie user didn't even think of a hot key.

As long as you guys are sure it's just a scripting issue on that button, I'll proceed to enter in all the other Projects.

For what it is worth - The one chap that tried to help me on the xTuple forum confirmed it is reproduced on his set up also.

**Comment by szuke** [comment]
Sorry, didn't mean to come off harsh. I created my 20 projects by using the Copy Project context menu.

**Comment by comtineo01** [comment]
andersdd - Hot Key working so far. I'll give a shout if it gets hinky after 15 or so entries also. Thanks!

szuke - "I created my 20 projects by using the Copy Project context menu" - If we were using two different methods of creating them wouldn't that be the reason I found the issue and you didn't ?

Also - Couldn't find any 'Copy' commands in the Projects menus and windows....what does that mean 'context menu' ?     Thank you for the follow up, helped me understand.
